core/src/main/java/com/sequenceiq/cloudbreak/service/stack/flow/StackOperationService.java

```diff
@@ -171,6 +180,7 @@ public FlowIdentifier stopInstances(Stack stack, Collection<String> instanceIds,
                 String instanceGroupName = entry.getKey();
                 int scalingAdjustment = entry.getValue().size() * -1;
                 updateNodeCountValidator.validateScalabilityOfInstanceGroup(stack, instanceGroupName, scalingAdjustment);
+                updateNodeCountValidator.validateInstanceGroupForStopStart(stack, instanceGroupName, scalingAdjustment);
```

Comment:
Stopping instances should not be allowed even with the forced flag. This check needs to be outside of the forced flag.

Reply:
Acknowledged, will change the location of this validation